src/app/(nosMissions)/education/page.js

```
@@ -242,7 +243,7 @@ export default function Education() {
           <div className="max-w-6xl mx-auto text-center">
             <motion.div variants={fadeInUp}>
               <h1 className="p-4 text-2xl md:text-4xl text-gray-950 font-bold text-center mb-8 relative">
-                SUIVI ET ENCADREMENT DE L&apos;ENFANT
+                SUIVI ET ENCADREMENT DE L'ENFANT
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)